library/ecp.c

```diff
@@ -1784,14 +1788,18 @@ int mbedtls_ecp_check_privkey( const mbedtls_ecp_group *grp, const mbedtls_mpi *
 #if defined(ECP_MONTGOMERY)
     if( ecp_get_type( grp ) == ECP_TYPE_MONTGOMERY )
     {
-        /* see [Curve25519] page 5 */
+        /* see [draft-irtf-cfrg-curves] page 7 */
```

Comment:
I'd prefer section references, including section name, instead of page references.